components/accordion/index.css

```diff
@@ -11,18 +11,135 @@ governing permissions and limitations under the License.
 */
 
 /* https://git.corp.adobe.com/Spectrum/spectrum-origins/pull/60 */
+
+.spectrum-Accordion {
+  --spectrum-accordion-item-height: var(--spectrum-component-height-200);
+  /*--spectrum-accordion-width: var(--spectrum-accordion-minimum-width);*/
+  --spectrum-accordion-disclosure-indicator-height: var(--spectrum-component-height-100);
+  /*--spectrum-accordion-disclosure-icon-asset: var(--spectrum-chevron100);*/
+  /*--spectrum-accordion-disclosure-indicator-to-text-space: var(--spectrum-accordion-disclosure-indicator-to-text);*/
```

Comment:
Once you've got the `<svg>` in your markup (see my previous comment about the icons), you'll be able to use this custom property to provide the space between the icon and the text.